composer.json

```diff
@@ -124,7 +124,8 @@
         "symfony/var-dumper": "^4.3",
         "symfony/web-profiler-bundle": "^4.3",
         "symfony/web-server-bundle": "^4.3",
-        "handcraftedinthealps/zendsearch": "^2.0"
+        "handcraftedinthealps/zendsearch": "^2.0",
+        "ext-intl": "*"
```

Comment:
Wondering if we should only make that a suggested dependency... Because this might break existing systems, and some of them won't even need it...

Reply:
@danrot but the tests will break if no ext-intl is installed. That's why I included it.

Reply:
Then you could also add it as a `dev` requirement. And isn't there another transliterator we could use, without requiring this PHP extension? Might be worth a thought, since it would not be a BC break then.

Reply:
@danrot it is already in `require-dev`, not `require`. I didn't searched for other transliterators, but I think PHP extensions are good enough and all other will be unefficient.